recipes/squashfskit-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "squashfskit" %}
+{% set version = "4.14" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  #- url: https://github.com/squashfskit/squashfskit/archive/v{{ version }}.tar.gz
+  #  fn: {{ name }}-v{{ version }}.tar.gz
+  #  sha256: 10ad288a4743a22ec69bd04bb5633b29072e4646c0062fd484e82ef247669dc7
+  - git_url: https://github.com/squashfskit/squashfskit
```

Comment:
yeah I think this is kind of a blocker for me

Reply:
I guess that the final version of this recipe will point to a release or specific commit. This is still work in progress, right @mingwandroid?

Reply:
Well, I don't want to release a binary where the copyright in the source code has been removed in this fashion, so while I'd love to release builds based on `sqashfskit` instead of sqashfs-tools (because there are a *load* of improvements, and it works on macOS too) I cannot contemplate a release as things stand as, to me, this is a clear violation of the moral rights of the orignal author.

We could apply a reversion of the commit that removed the original author's copyight attribution I suppose, but to me that would be a little bit 'wet'.

Not having builds of `squashfs` that work on macOS prevent people (if anyone is so interested!) from doing raspi conda hacking stuff (ok, in the way I'm currently mucking about with it), but of course I can always upload `squashfskit` builds to my own channel in the meantime (and have done so, though I feel a little bad about it).

I *had* hoped this would all be resolved nicely by now too. Time heals all wounds?